dojo/tools/anchore_grype/parser.py

```diff
@@ -23,44 +22,143 @@ def get_findings(self, file, test):
         data = json.load(file)
         dupes = dict()
         for item in data.get("matches", []):
-            cve = item["vulnerability"]["id"]
-            severity = self._convert_severity(item["vulnerability"]["severity"])
-            purl = PackageURL.from_string(item["artifact"]["purl"])
-            description = ""
-            description += f"\n**CVE:** {cve}"
-            if type(item["matchDetails"]) is dict:
-                description += f'\n**Matcher:** {item["matchDetails"]["matcher"]}'
+            vulnerability = item['vulnerability']
+            vuln_id = vulnerability["id"]
+            vuln_namespace = vulnerability.get('namespace', None)
+            vuln_datasource = vulnerability.get('dataSource', None)
+            vuln_severity = self._convert_severity(vulnerability["severity"])
+            vuln_urls = vulnerability.get('urls', None)
+            vuln_description = vulnerability.get('description', None)
+            vuln_fix_versions = None
+            fix = vulnerability.get('fix', None)
+            if fix:
+                vuln_fix_versions = fix.get('versions', None)
```

Comment:
```suggestion
            if 'fix' in vulnerability:
                vuln_fix_versions = vulnerability.get('fix').get('versions')
```
Better, you can do it in one line:
```
vuln_fix_versions = vulnerability.get('fix', []).get('versions')
```
But the previous line is more readable.

Reply:
done